projects/js-packages/publicize-components/src/components/admin-page/toggles/social-module-toggle/index.tsx

```diff
@@ -92,7 +92,8 @@ const SocialModuleToggle: React.FC = () => {
 				{ ! is_wpcom
 					? __(
 							'When enabled, you’ll be able to connect your social media accounts and send a post’s featured image and content to the selected channels with a single click when the post is published.',
-							'jetpack-publicize-components'
+							'jetpack-publicize-components',
+							/* dummy arg to avoid bad minification */ 0
```

Comment:
TS won't be happy with this. A better solution will be to use `__` in one string and `_x` in the other with some optional/empty context.